cpp/oneapi/dal/graph/detail/undirected_adjacency_vector_graph_impl.hpp

```diff
@@ -129,6 +129,14 @@ class ONEDAL_EXPORT undirected_adjacency_vector_graph_impl {
         _topology.set_topology(std::forward<Args>(args)...);
     }
 
+    inline void set_edge_values(EdgeValue* values, int64_t values_count) {
```

Comment:
What is it?

Reply:
It was added to support read of undirected weighted graphs